scripts/sct_label_utils.py

```diff
@@ -598,6 +602,29 @@ def launch_sagittal_viewer(self, labels):
 
         return output
 
+    def remove_specific_labels(self, labels, isKeep = False):
```

Comment:
same here, why not `remove_labels` for function name?

Reply:
Sounds good! Changed in commit [59747a7](https://github.com/neuropoly/spinalcordtoolbox/commit/59747a751482a08228783fec1cbc9623d988a3d9)